### ISSUE TITLE
Modder Permissions

### ISSUE THREAD

**Issue by calclavia** [calclavia/Voltz-Mod-Pack#1]
Please write your permission here if you allow your mod to be included in the Voltz Mod Packin the Technic Launcher and FTB launcher 

About the Technic Team: Note that this pack is not going to be part of the Technic pack but will be a separate pack, simply managed by that team.

About the FTB Pack: The FTB version of this pack will be launched as a third party pack. and manily managed by cammygames

Permission will need to be received from each author who allows their mod to be included in this pack. I will monitor and make sure every single mod going into this pack have received permission from their authors.

**Comment by calclavia** [comment]
I, Calclavia hereby give the Voltz Mod Pack permission to distribute four of my mods being Basic Components, Atomic Science, ICBM and Assembly Line given that I receive proper credit.

**Comment by CovertJaguar** [comment]
I give the Voltz pack permission to include Railcraft. This does NOT apply  to Technic/Tekkit or anything else. You must follow the requires as listed on the Railcraft Wiki. You must provide a publicly posted and easy to find list of permissions from all the mod authors that require such things. If a mod licence allows you to include a mod without permission, please link to the license instead.

**Comment by aidancbrady** [comment]
Yes yes, you're free to use Mekanism in this pack.

**Comment by ghost** [comment]
I, Mattredsox give Calclavia permission to use Electric Expansion in this pack according to the terms on our git.

**Comment by XCompWiz** [comment]
I hereby grant Voltz Mod Pack permission to include Mystcraft provided no profit is made from its inclusion or from the pack in general and it gives proper credit to all mod authors.

**Comment by ghost** [comment]
i  give you my Permission to add MFFS to Voltz Mod Pack

**Comment by DarkGuardsman** [comment]
I've already said it before, and next time you need it you can just Quote me saying it. You, Calclaiva, have permission to use any of my mods using UE in your mod pack. This includes Steam Power, Basic Pipes, Auto Sentries, and Mining machines when updated.

**Comment by Eearslya** [comment]
The Voltz modpack is hereby allowed to include PowerTools provided it remains exactly as it is distributed and is properly credited.

**Comment by DarkGuardsman** [comment]
if you want you can also include TheCowGod's GreaterSecurity mod since i do most of the development on it. Also he said i can do what ever i want with the mod :)

https://dl.dropbox.com/u/70622753/updater/.minecraft/mods/GreaterSecurityTestMC1.4.5A1.zip

**Comment by calclavia** [comment]
As much as we want to include your mods @Darkguardsman, the last time I played with them they're pretty bugged, but I'm not sure how they are currently. All mods in this mod pack are going to be distributed to tons of players so we must make sure the mod is at least crash-free. All of them must be updated to MC 1.4.6. It'd be great if you can get on the IRC more often so I can talk with you about stuff.

**Comment by DarkGuardsman** [comment]
I get on irc once in a while but no one is on. GreaterSecurity has no major issue and a quick test should show that. Also once again why does everyone update for a version of minecraft that might not come out yet? But if needed i can have it updated for 1.4.6. Sad just got all my repo's rebuilt for 1.4.5 and was getting ready to finish the updates.

**Comment by ghost** [comment]
@calclavia  Update MFFS for 1.4.6 and UE BasicComponents_v1.2.1.306  

 https://www.dropbox.com/s/fey14kr4nv96vr9/ModularForceFieldSystemV2.2.8.2.7.zip

**Comment by dries007** [comment]
I give the Voltz Mod Pack permission to distribute versions of ForgeEssentials if they are downloaded from http://files.minecraftforge.net/ForgeEssentials/ or otherwise instructed in the topic of our IRC channel #forgeessentials.

**Comment by JeffPerando** [comment]
Yes, you may use Hawk's Machinery in the Voltz Mod Pack...

**Comment by flowtender** [comment]
um I don't know why I'm posting this since my license is explicitly 'do what you want' but here is my permission anyway I guess

p.s. if you are any other person reading this and you want to make a mod pack with my mod in it GO AHEAD I don't care

**Comment by beerik94** [comment]
Yes, when Biotech is ready you may use it in the Voltz/UE mod pack

**Comment by AlgorithmX2** [comment]
I give permissions to use Applied Energistics in the voltz pack, even tho you don't need it cause my license already says you can...

**Comment by flowtender** [comment]
i know right!!!

**Comment by GregoriusT** [comment]
I give permission to use the GregTech-Addon in the Voltz Modpack. This however, doesn't include the TechnicLauncher itself.

**Comment by Rushmead** [comment]
I Give Full Permission To Allow Voltz To Include Minechem in it
